src/app/maps/page.tsx

```diff
@@ -1,18 +1,37 @@
 'use client';
 
-import React from 'react';
+import React, { useEffect } from 'react';
 import Map from '@/components/map/Map';
 import getLocation from '@/hooks/useLocation';
 import BottomNav from '@/components/bottom-nav/BottomNav';
+import getSightingsByLocation from '@/hooks/getSightingsByLocation';
 
 const Maps: React.FC = () => {
   const {
     latitude: currentLatitude,
     longitude: currentLongitude,
-    //callCount,
+    // callCount,
   } = getLocation({ onLocationUpdate: () => {} });
 
-  const apiKey = ' '; // (grab form discord) a story has been created to find a better approach
+  const apiKey = ' '; // (grab from discord) a story has been created to find a better approach
+
+  // Fetch sightings using useEffect for testing purposes
+  useEffect(() => {
+    const fetchSightings = async () => {
+      try {
+        const sightings = await getSightingsByLocation(
+          -73.985428,
+          40.748817,
+          1
+        );
+        console.log('Fetched sightings:', sightings);
+      } catch (error) {
+        console.error('Error fetching sightings:', error);
+      }
+    };
```

Comment:
rebase main since your gonna be able to do this now, but use `useQuery` and pass in the fetch sightings function as a callback within `useQuery`. That hook will have all of the loading, error and other important vars built in with minimal extra code for you.

[Link to use query documentation](https://tanstack.com/query/latest/docs/framework/react/reference/useQuery)

Reply:
u can also see example implementation in the car rental fe repo too

Reply:
is this change for the Maps page or the Hook? because this Maps page code is only test code and not meant to get merged. But I'll def keep note for my maps FE story